cpp/docs/DEVELOPER_GUIDE.md

```diff
@@ -1169,7 +1169,7 @@ instance of a class object to represent a null string.
 
 The `string_view` contains comparison operators `<,>,==,<=,>=` that can be used in many cudf
 functions like `sort` without string-specific code. The data for a `string_view` instance is
-required to be [UTF-8](#UTF-8) and all operators and methods expect this encoding. Unless documented
+required to be [UTF-8](#utf-8) and all operators and methods expect this encoding. Unless documented
```

Comment:
Same question, where is this actually pointing to?

Reply:
Yes. regardless of case, the link points to right location. `xrefcheck` tool expected the link to be in smaller case!

Reply:
Not sure how I missed this one, thanks.